python/lsst/sconsUtils/state.py

```diff
@@ -345,22 +345,22 @@ def ClassifyCc(context):
         env.Append(LINKFLAGS='--coverage')
 
     #
-    # Enable C++11 support (and C99 support for gcc)
+    # Enable C++14 support (and C99 support for gcc)
```

Comment:
Is this C99 comment still relevant?

Reply:
I don't know. But upping the C version was not part of the RFC.

Reply:
Right. I'll leave the comment as is.